### ISSUE TITLE
Upgrade Restart Error

### ISSUE THREAD

**Issue by 197788** [ipfs-shipyard/ipfs-desktop#1578]
Would not upgrade to 0.12.1 with doing a restart. Had to do a full shutdown. Then complained about the ipfs.exe being gone from where it was, do i want to delete the shortcut.

Please describe what you were doing when this error happened.

**Specifications**

- **OS**: win32
- **IPFS Desktop Version**: 0.12.1
- **Electron Version**: 8.2.5
- **Chrome Version**: 80.0.3987.165

**Error**

```
TypeError: ctx.checkForUpdates is not a function
    at click (C:\Users\jason\AppData\Local\Programs\IPFS Desktop\resources\app.asar\src\tray.js:198:30)
    at MenuItem.click (electron/js2c/browser_init.js:1501:9)
    at Function.executeCommand (electron/js2c/browser_init.js:1762:13)
```

**Comment by jessicaschilling** [comment]
@olizilla, could you kindly triage? Thank you!

**Comment by 197788** [comment]
Well, it's fixed now because I downloaded 11.4 from start, reinstalled, and then it did do the upgrade after restarting. So I suppose you can close it. Thanks.

Jul 13, 2020 10:23:47 PM Jessica Schilling <notifications@github.com>:
